notifier.rst

```diff
@@ -0,0 +1,568 @@
+.. index::
+   single: Notifier
+
+Creating and Sending Notifications
+==================================
+
+.. versionadded:: 5.0
+
+    The Notifier component was introduced in Symfony 5.0 as an
+    :doc:`experimental feature </contributing/code/experimental>`.
+
+Current web applications use many different channels to send messages to
+the users (e.g. SMS, Slack messages, emails, push notifications, etc.). The
+Notifier component in Symfony is an abstraction on top of all these
+channels. It provides a dynamic way to manage how the messages are send.
+Get the Notifier installed using:
+
+.. code-block:: terminal
+
+    $ composer require symfony/notifier
+
+Channels: Chatters, Texters, Email and Browser
+----------------------------------------------
+
+The notifier component can send notifications to different channels. Each
+channel can integrate with different providers (e.g. Slack or Twilio SMS)
+by using transports.
+
+The notifier component supports the following channels:
+
+* `SMS <SMS Channel>`_ sends notifications to phones via SMS messages
+* `Chat <Chat Channel>`_ sends notifications to chat services like Slack
+  and Telegram;
+* `Email <Email Channel>`_ integrates the :doc:`Symfony Mailer </mailer>`;
+* Browser uses :ref:`flash messages <flash-messages>`.
+
+.. tip::
+
+    Use :doc:`secrets </configuration/secrets>` to securily store your
+    API's tokens.
+
+.. _notifier-texter-dsn:
+
+SMS Channel
+~~~~~~~~~~~
+
+The SMS channel uses :class:`Symfony\\Component\\Notifier\\Texter` classes
+to send SMS messages to mobile phones. This feature requires subscribing to
+a third-party service that sends SMS messages. Symfony provides integration
+with a couple popular SMS services:
+
+=======  ===========================  ========================================
+Service  Package                      DSN
+=======  ===========================  ========================================
+Twilio   ``symfony/twilio-notifier``  ``twilio://SID:TOKEN@default?from=FROM``
+Nexmo    ``symfony/nexmo-notifier``   ``nexmo://KEY:SECRET@default?from=FROM``
+=======  ===========================  ========================================
+
+To enable a texter, add the add the correct DSN in your ``.env`` file and
```

Comment:
Typo: `add the` is doubled :)

Reply:
Fixed in latest PR ✌🏻 Thanks for the hint